GSA-TTS/FAC
# Add dissemination API validator and schemas

I picked up @jadudm's experimentation branch, added schemas for the remaining views, and made some small changes to the validator's logic. This is still pretty rough, but gives us a starting point to work from. 

How to test locally:
1. Fire up the dockerized app with `docker compose up`.
2. Pick a view to validate and construct your command. For example:
    ```bash
    docker compose run web python manage.py validate_api --view vw_auditor
    ````
    a. You might want to write the validation results to a text file. There's an example, `validate_log.txt` in this draft PR.
3. All of the views (except `revision`) should be valid according to the schemas in this draft PR. You might try changing a schema around to see what kinds of failures are triggered.

@eazapata23 I left you some wayfinding comments inline since you'll be working on validation over the next couple of weeks.
